packages/desktop-client/src/hooks/useProperFocus.tsx

```diff
@@ -29,17 +35,29 @@ function focusElement(el, refocusContext) {
     el.focus();
   }
 
-  if (el.tagName === 'INPUT') {
+  if (el instanceof HTMLInputElement) {
```

Comment:
Another functional change. But I believe the logic will evaluate the same way. Just a bit stronger typechecking.